apps/mobile/src/store/keychains/stacks/stacks-keychains.read.ts

```diff
@@ -47,7 +47,9 @@ export function useStacksSigners() {
 }
 
 export function useStacksSignerAddresses() {
-  return useStacksSigners().list.map(signer => signer.address);
+  const { list } = useStacksSigners();
+
+  return useMemo(() => list.map(signer => signer.address), [list]);
```

Comment:
```suggestion
  const { stacksSigners } = useStacksSigners();

  return useMemo(() => stacksSigners.map(signer => signer.address), [stacksSigners]);
```

Reply:
Do you even need `useMemo` here at all? https://kentcdodds.com/blog/usememo-and-usecallback